Update errors-inbox.mdx (Publish ~May 9, 2022) (#7108)
newrelic/docs-website: src/content/docs/errors-inbox/errors-inbox.mdx

Using the dropdown in the top right, you can sort the list of grouped errors by the number of occurrences or by the error that was last seen (latest first).

### Bulk Actions 

> You can update the status (e.g. resolve, ignore, unresolve) of multiple error groups, up to 2,000, all at once. In the inbox view, you can select the Error group checkbox and update all error groups in the inbox. You can also select individual error groups and update their status.

Reviewer: ```suggestion
You can update the status (for example, resolve, ignore, unresolve) of multiple error groups (up to 2,000) all at once. In the inbox view, select the **Error group** checkbox to update all of your error groups in the inbox. You can also select individual error groups and update their statuses.
```
